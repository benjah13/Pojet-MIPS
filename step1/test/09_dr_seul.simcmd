#test la comande dr ; fonctionnement correct
dr $0
