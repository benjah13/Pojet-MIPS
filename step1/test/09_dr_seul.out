REGISTRE  $0 = 0
